sdk/security/keyvault/azadmin/backup/autorest.md

```diff
@@ -94,4 +96,9 @@ directive:
   - from: swagger-document
     where: $.definitions.SelectiveKeyRestoreOperationParameters
     transform: $["description"] = "Parameters for the selective restore operation"
+
+  # fix up span names
+  - from: client.go
+    where: $
+    transform: return $.replace(/StartSpan\(ctx, "Client/, "StartSpan(ctx, \"backup.Client");
```

Comment:
Do we need this here? Are we able to replace StartSpan(ctx, "client... with StartSpan(ctx, "backup.Client.... in client.go since this isn't modifying generated files? Or what is the benefit of adding this transform?

Reply:
For the key vault SDKs, we do directly expose the generated code albeit with some build-time transforms to get the shape we want. So, this transform does impact the generated code.

That said, I did this in an effort to disambiguate the client names in the traces. For this SDK there are three clients across three subpackages, each with the name `Client`. So, I modified the span name to include the package name.

Reply:
I will point out though that there are no common methods (at present) across the clients. So, one could make the argument that this is enough of a distinction that the package prefix isn't required. I prefer keeping it for being explicit but am open to options.

Reply:
I will also look into seeing if we can automatically do this in the code generator. I opened https://github.com/Azure/autorest.go/issues/1071 to track that.